content/en/docs/setup/install/multicluster/index.md

```diff
@@ -0,0 +1,724 @@
+---
+title: Multicluster Installation
+description: Install an Istio mesh across multiple Kubernetes clusters.
+weight: 30
+aliases:
+    - /docs/setup/kubernetes/multicluster-install/
+    - /docs/setup/kubernetes/multicluster/
+    - /docs/setup/kubernetes/install/multicluster/
+keywords: [kubernetes,multicluster]
+test: yes
+owner: istio/wg-environments-maintainers
+---
+When installing Istio across multiple clusters, there are more things to
+consider than with a [single cluster installation](/docs/setup/getting-started).
+
+Although there is no default installation procedure for a multicluster mesh, as it will
+be different based on your cluster and network architecture, this guide covers some
+of the most common concerns:
+
+- [Network topologies](/docs/ops/deployment/deployment-models#network-models):
+  one or two networks
+
+- [Control plane topologies](/docs/ops/deployment/deployment-models#control-plane-models):
+  multi-primary, primary-remote
+
+Before you begin, review the [deployment models guide](/docs/ops/deployment/deployment-models)
+which describes the foundational concepts used throughout this guide.
+
+## Requirements
+
+This guide requires that you have two Kubernetes clusters with any of the
+[supported versions](/docs/setup/platform-setup).
+
+If the clusters are on different networks (i.e. no direct connectivity between
+pods on different clusters), they must have ingress gateway services which are
```

Comment:
Make it clear that those can be set up using Istio after Istio is installed.

Otherwise this reads like they need to set up ingress gateways before installing Istio.

Alternatively just drop this from here and talk about it when configuring the second cluster on a different network.

Reply:
I've removed this and added some basic text to the multicluster sections.